src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl

```diff
@@ -661,6 +661,15 @@
               "type": "string"
             }
           },
+          {
+            "in": "header",
+            "name": "lang",
+            "required": true,
+            "description": "Language requested by the user",
+            "schema": {
+              "type": "string",
+            }
+          },
```

Comment:
the `lang` header should be add also for POST /transactions/{transactionId}/auth-requests api call or POST /transactions one to propagate user language to transactions-service b.e.?

POST sessions api call is performed for guest cards payment only so this information have to be propagated also for transactions performed with other methods than cards